Dockerfile

```diff
@@ -15,15 +15,20 @@ COPY tailwind.config.ts .
 COPY postcss.config.js .
 COPY docker-entrypoint.sh .
 
-# We replace NEXT_PUBLIC_* variables here with placeholders
-# as next.js automatically replaces those during building
-# Later the docker-entrypoint.sh script finds such variables and replaces them
-# with the docker environment variables we have set
-RUN NEXT_PUBLIC_MEMPOOL_API=APP_NEXT_PUBLIC_MEMPOOL_API \
-    NEXT_PUBLIC_API_URL=APP_NEXT_PUBLIC_API_URL \
-    NEXT_PUBLIC_NETWORK=APP_NEXT_PUBLIC_NETWORK \
-    NEXT_PUBLIC_DISPLAY_TESTING_MESSAGES=APP_NEXT_PUBLIC_DISPLAY_TESTING_MESSAGES \
-    npm run build
+
+ARG NEXT_PUBLIC_MEMPOOL_API
```

Comment:
Do we need to name the args the same way as the env variables? For example, why not just `MEMPOOL_API` for this one?

Reply:
I used `NEXT_PUBLIC_*` for ARGS to match with the [.env.example](https://github.com/babylonlabs-io/simple-staking/blob/dev/.env.example) for consistency